lib/solid_queue/supervisor/pidfile.rb

```diff
@@ -29,7 +29,7 @@ def check_status
         else
           FileUtils.mkdir_p File.dirname(path)
         end
-      rescue Errno::ESRCH => e
+      rescue Errno::ESRCH => _e
```

Comment:
It seems that removing unused variables makes the code look simpler.

```suggestion
      rescue Errno::ESRCH
```

Reply:
It makes sense.